autopush/router/webpush.py

```diff
@@ -20,6 +20,8 @@
 from autopush.router.interface import RouterException, RouterResponse
 from autopush.router.simple import SimpleRouter
 
+TTL_URL = "https://webpush-wg.github.io/webpush-protocol/#rfc.section.6.2"
```

Comment:
may want to toss these into a central file, only so we don't have to chase them down when specs change.